### DISCUSSION TITLE
TLoZ Majora's Mask: The character turns black or colored next to a fire.

### DISCUSSION THREAD

**Discussion by Xernobyw** [mupen64plus-ae/mupen64plus-ae#969] [Q&A]
Hi, I'm having trouble with the TLoZ Majora's Mask game. When I approach Link or another character near a fire, he turns black or colored, as shown in the images. But when I walk away, the colors go back to normal. I don't know why this happens. I'm using the "Glide64-Fast" plugin, my phone is a Moto G7 Play (Android 10), and I'm using the app version 3.0.293 (beta)-free.

Glitch:
![Screenshot_20211104-195250~2](https://user-images.githubusercontent.com/93743009/140432839-dd4f5a9f-e0a5-49da-a88e-5eeeb34d437c.png)
![Screenshot_20211104-195230~2](https://user-images.githubusercontent.com/93743009/140432847-3672c21d-391c-403c-b4b1-82439c073c67.png)
![Screenshot_20211104-194603~2](https://user-images.githubusercontent.com/93743009/140432857-76e0271f-d1c9-46cf-a3b7-ad592150cbf8.png)
![Screenshot_20211104-194709~2](https://user-images.githubusercontent.com/93743009/140432874-d706ecf5-5e38-4d79-bd0d-3ec2c6878a7a.png)
![Screenshot_20211104-194827~2](https://user-images.githubusercontent.com/93743009/140432893-fddb6361-c830-4ba5-afdb-9da4909332d0.png)

Normal:
![Screenshot_20211104-194848~2](https://user-images.githubusercontent.com/93743009/140432971-52543994-8298-458b-83e6-6a46631f3740.png)

**Comment by fzurita** [comment]
Hmm, interesting. It's possible that the recent neon optimizations are causing this. I'll have to take a look.

**Comment by fzurita** [comment]
In the mean time, you can select the game, go to settings, and change the emulation profile to GLideN64-Medium.

**Reply by Xernobyw** [reply]
Okay

**Comment by fzurita** [comment]
I published am update to the play store. It should have the fix once it's approved.

**Comment by Xernobyw** [comment]
Okay. I downloaded the update and it fixed the color and lighting issue.
Thank you ^^

Em sáb, 6 de nov de 2021 13:03, Francisco Zurita ***@***.***>
escreveu:
